packages/angular/cli/models/schematic-command.ts

```diff
@@ -17,10 +17,12 @@ import {
 } from '@angular-devkit/core';
 import { NodeJsSyncHost } from '@angular-devkit/core/node';
 import { DryRunEvent, UnsuccessfulWorkflowExecution, workflow } from '@angular-devkit/schematics';
+import { standardFormats } from '@angular-devkit/schematics/src/formats';
```

Comment:
Why the deep import?

Reply:
Ups, good catch. Didn't see it's already exported from `@angular-devkit/schematics`.